### PR TITLE
Update README to state its  community maintained

### PR DESCRIPTION
**Pull request by gernest**
I'm neither employed nor paid by twilio to maintain this.

If you are here, please see related #36

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -555,7 +555,7 @@ for the full interface documentation.
 
 ## Maintenance
 
-The project is hosted and maintained by Twilio; we welcome external contributors
+The project is hosted and maintained by the  open source community; we welcome external contributors
```
Comment (fpetkovski) [inline README.md:558]:
```suggestion
The project is hosted and maintained by the open source community; we welcome external contributors
```

**Review by fpetkovski — APPROVED** [review]
LGTM, would just like to know if @kevinburke is okay with this.

File: README.md [hunk 2]
```diff
@@ -555,7 +555,7 @@ for the full interface documentation.
 
 ## Maintenance
 
-The project is hosted and maintained by Twilio; we welcome external contributors
+The project is hosted and maintained by the open source community; we welcome external contributors.
```
Comment (kevinburkesegment) [inline README.md:558]:
```suggestion
While initial design and development occurred at Twilio Segment, the project is now maintained by the open source community. We welcome external contributors.
```

**Review by kevinburkesegment — APPROVED** [review]

**Comment by kevinburkesegment** [comment]
One suggestion. Thank you!

**Pull request merged**